Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/cxf/CXFSAMLCallbackHandler.java

```diff
@@ -51,15 +55,18 @@
 public class CXFSAMLCallbackHandler implements CallbackHandler {
 
     private static final Logger LOG = LoggerFactory.getLogger(CXFSAMLCallbackHandler.class);
-
+    private PropertyAccessor accessor;
     public static final String HOK_CONFIRM = "urn:oasis:names:tc:SAML:2.0:cm:holder-of-key";
     private HOKSAMLAssertionBuilder builder = new HOKSAMLAssertionBuilder();
+    private Crypto issuerCrypto = null;
 
     public CXFSAMLCallbackHandler() {
+        accessor = PropertyAccessor.getInstance();
     }
 
-    public CXFSAMLCallbackHandler(HOKSAMLAssertionBuilder builder) {
+    public CXFSAMLCallbackHandler(final HOKSAMLAssertionBuilder builder) {
         this.builder = builder;
+        accessor = PropertyAccessor.getInstance();
```

Comment:
To prevent code duplication, would it be possible to change the body of this constructor to the following?

    this();
    this.builder = builder;